pkg/dockerregistry/server/manifesthandler/manifestschema2handler.go

```diff
@@ -80,8 +82,33 @@ func (h *manifestSchema2Handler) verifyLayer(ctx context.Context, fsLayer distri
 		return errUnexpectedURL
 	}
 
-	desc, err := h.blobStore.Stat(ctx, fsLayer.Digest)
-	if err != nil {
+	// XXX https://bugzilla.redhat.com/show_bug.cgi?id=1745743
+	// Due to the "eventual consistent" nature of some of image registry
+	// storage driver backends we have to do here a few retries. We gonna
+	// need to address this properly in the future. Here we may take up to
+	// three seconds to report a blob as unknown.
+	var desc distribution.Descriptor
```

Comment:
Nit - clean up the comment to be more concise. Ex:

```
// https://bugzilla.redhat.com/show_bug.cgi?id=1745743
// AWS S3 (and potentially other object stores) only have eventual consistency guarantees.
// Stat can fail here if an image layer was recently pushed.
// In the event Stat returns `ErrBlobUnknown`, retry up to 3 seconds.
```

Reply:
done.